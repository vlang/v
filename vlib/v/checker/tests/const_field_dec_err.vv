const a = 1

fn main() {
	a--
}
