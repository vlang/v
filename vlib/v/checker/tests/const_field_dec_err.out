vlib/v/checker/tests/const_field_dec_err.vv:4:2: error: cannot modify constant `a`
    2 | 
    3 | fn main() {
    4 |     a--
      |     ^
    5 | }
